lib/dpkg.sh

```diff
@@ -132,7 +132,21 @@ dpkg_Sy() {
 }
 
 dpkg_Ss() {
-  apt-cache search "$@"
+  IFS=$'\n' packages=($(apt-cache search "$1"))
+  if [[ ${#packages[@]} -ne 0 ]]
+  then 
+    for package in ${packages[@]}
+    do
+      name=${package%% - *}
+      desc=${package#* - }
+      dpkg-query -W "$name" > /dev/null 2>&1
+      if [[ $? -eq 1 ]]; then
+        echo -e "package/$name \n    $desc"
```

Comment:
nit: don't use `echo -e`, use `printf` instead.

Reply:
@NgoHuy `${packages[@]:-}` is shorter (`for` won't do anything on empty list)